Victoria - bp557/victoria_peng_week10/Victoria_Peng_Week10.do

```diff
@@ -0,0 +1,112 @@
+********************************************************************************
+* Econ 490: Week 10
+* Handling datasets in Stata
+* Bingjie (Victoria) Peng
+* Nov 11st, 2022
+********************************************************************************
+clear 
+set seed 1
+set more off
+global data_location = "/Users/victoriapeng/Desktop/490 Research Field/Week10/03_assignment/01_data" 
+********************************************************************************
+
+*1
+clear
+import excel "CIV_populationdensity", sheet("Population density") firstrow clear // import the excel dataset
```

Comment:
You can set all your file paths as globals at the start of the document potentially.